src/config/tours.js

```diff
@@ -0,0 +1,70 @@
+import {
+  PHASE_DRAWER,
+  SPACE_PREVIEW_ICON_CLASS,
+  SPACE_SCREEN,
+  VISIT_SPACE_BUTTON,
+  VISIT_SPACE_INPUT,
+  SETTINGS_ACTIONS_CLASS,
+  LANGUAGE_SELECT_ID,
+  STUDENT_MODE_SWITCH_ID,
+} from './selectors';
+
+const tours = {
+  VISIT_SPACE_TOUR: 'visitSpace',
+  SETTINGS_TOUR: 'settings',
+};
+
+const VISIT_SPACE_TOUR_STEPS = [
+  {
+    target: `.${VISIT_SPACE_BUTTON}`,
+    title: 'Welcome to Graasp Desktop!',
```

Comment:
You will need to use `i18n.t` to translate the texts.

Reply:
yes definitely. I think we should do that next. Refine the English statements and provide the french alternatives.